README.md

```diff
@@ -12,7 +12,28 @@
 
 - **Tech stack:** The technology stack used for this project is known as the MERN stack, which consists of MongoDB, ExpressJS, React, and NodeJS.
 
-## R2.	Dataflow Diagram
+## R2.	Dataflow Diagram 
+
+Dataflow diagram will help to have a visual representation of the flow of information in the application. It includes using various symbols to represent processes, data stores, external entities and data flow. 
+
+**1. Process:**
+
+It represents the logical processing of input data to produce and output data.
+
+**2. Data store:**
+
+It represents the data storage of inputs received from users that can be used for later use for retracting user information.
+
+**3. External entities:**
+
+These represent any flow of information or data from any other external organisation, application programming interface or system. These usually represent the edges of the DFD diagrams.
+
+**4. Data flows:**
+
+The data flow lines represent the input and output and the flow of data through processes, external entities and data stores.
+
+![dfd](./dfd.png)
+
```

Comment:
We would want different DFDs for:

 * User management
 * Exercise list management
 * User recording sets, reps of exercises

I recommend for the exercise list doing a bulk import with a npm task

Reply:
ok Thanks, will work on that.

Reply:
Ji and I had a discussion to go with single user role where user manages his own account with login and logout access only. The exercise list process ultimately merges with the main app data flow once the data is imported.